tests/accidental_tests.ts

```diff
@@ -286,19 +281,107 @@ function basic(options: TestOptions): void {
   options.assert.ok(true, 'Full Accidental');
 }
 
+function genAccidentals(): string[] {
+  const accs = [ '#', '##', 'b', 'bb', 'n', '{', '}', 'db', 'd', '++', '+', '+-'];
+  accs.push('bs', 'bss', 'o', 'k', 'bbs', '++-', 'ashs', 'afhf');
```

Comment:
Super minor question. Why do we push a bunch of other items on the second line?
Why not stick it all in the original array? You can always add a // comment if you want to break it up into two lines.

Reply:
If I put it in one line, prettier will change the arrengement to:
```js
const accs = [ '#', 
    '##', 
    'b',
    ....
```
This increases the number of required lines a lot.

Reply:
OK, this is just in the tests file, so I don't care much. But generally we shouldn't worry about how prettier formats our files. But I've had some luck in adding extra comments // to break up some lines to avoid prettier edge cases. :-)